### DISCUSSION TITLE
ext-remover.net

### DISCUSSION THREAD

**Discussion by WalkTheEarth** [3kh0/ext-remover#1330] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi! I have a question regarding `ext-remover.net`, i would like to know if anyone has the source code to that. i want to host it on my github pages, so i can browse it without exthang3r failing and re-enabling lightspeed every 10 mins

**Comment by no-name-6655** [comment, marked as answer]
view-source:ext-remover.net

**Comment by AshtonDavies** [comment]
The site source is not open yet, but you can save pages for local use or use Blobby's official data URL or something.
